recipe/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "landingai" %}
+{% set version = "0.3.49" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/landingai-{{ version }}.tar.gz
+  sha256: 9bc8051f3ecf2b686b285b03967cc42c47931dae62f54dafd5a23be7c95fb2b3
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  # segmentation-mask-overlay not avaliable
+  skip: True  # [s390x]
+  # Unsatisfiable dependencies for platform osx-arm64: {MatchSpec("tqdm==4.64.0=py310hca03da5_0") py313
+  # same for cryptography on py312
+  skip: True  # [py>311]
+  # PyPi Requires: Python <4.0, >=3.9
+  skip: True  # [py<39]
+
+requirements:
+  host:
+    - python
+    - poetry-core
+    - pip
+  run:
+    - python
+    - opencv >=4.5,<5.0
+    - numpy >=1.21.0,<2.0.0
+    - pillow >=9.0,<11.0
+    - pydantic 2.*
+    - requests 2.*
+    - urllib3 >=1.26.0,<2.0.0
+    - bbox-visualizer >=0.1.0,<0.2.0
+    - segmentation-mask-overlay >=0.3.4,<0.4.0
+    - imageio 2.*
+    - aiohttp >=3.7.3,<4.0.0
+    - aiofiles >=0.7.0,<1.0.0
+    - tqdm >=4.64.0,<5.0.0
+    - tenacity >=8.2.3,<9.0.0
+    - pandas 2.*
+    - requests-toolbelt >=1.0.0,<2.0.0
+    - pydantic-settings >=2.3.1,<3.0.0
+  run_constrained:
+    - snowflake-connector-python >=3.10.0,<4.0.0
+    - cryptography >=39.0.0,<40.0.0
```

Comment:
The source code says `cryptography = {version="^39.0.0", optional = true}`, but the current version is 43.0.3. I would relax the pinning as >=39.0.0 because this kind of versioning is the edge case, or at least `cryptography >=39.0.0,<50.0.0`.

FYI, CVE-2024-26130 fixed only in `cryptography >=42.0.4`